src/pages/home/Home.jsx

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import "./Home.css";
+export function Home() {
+  return (
+    <div className="grid-layout">
+      <main className="home-main">
+        <div className="home-main-child">
+          <h1 className="ft-w-900">Quiz</h1>
+          <p className="ft-w-500 ft-grey">Lets play quiz!</p>
+          <a href="#quiz" className="btn btn-primary btn-ls">
+            Explore
+          </a>
+        </div>
+        <div className="home-main-child">
+          <img
+            src="https://raw.githubusercontent.com/mikesch803/fit-quiz/d4b0d07666c735b7d8304c2c519158c8b38e7efc/assests/images/Questions-cuate.svg"
+            alt="boy-illustration"
+            className="img-resp"
+          />
+        </div>
+      </main>
+      <section className="home-section">
+        <h1 className="ft-w-700 txt-center" id="quiz">
+          Categories
+        </h1>
+        <ul className="categories-list">
+          {Array(3)
```

Comment:
As far as I understood it. You are making 3 arrays with the given data right ?

Reply:
so i used the dummy data as of now, when i will work on functionality, i will change this